docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,61 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+To create clustered caches, you will need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
+
+=== Creating cache manager with clustering capabilities
```

Comment:
Before this point, there needs to be minimal documentation on starting a Terracotta server so that the client can connect to it.